android/gradle.properties

```diff
@@ -16,3 +16,7 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+MYAPP_RELEASE_STORE_FILE=my-release-key.keystore
+MYAPP_RELEASE_KEY_ALIAS=my-key-alias
+MYAPP_RELEASE_STORE_PASSWORD=sw]8=l}(A+(5Nt(0US-d
+MYAPP_RELEASE_KEY_PASSWORD=sw]8=l}(A+(5Nt(0US-d
```

Comment:
is it ok to store passwords in config file?

Reply:
Also, maybe it make sense to rename those variables to something like `PTUSHKI_KEY_ALIAS`?

Reply:
@GlebShaucou any updates here?